### DISCUSSION TITLE
latency problems

### DISCUSSION THREAD

**Discussion by BenClam** [GrandOrgue/grandorgue#1079] [Help]
[screen-latency-BDeschamps.docx](https://github.com/GrandOrgue/grandorgue/files/8195650/screen-latency-BDeschamps.docx)
I have problem of latency with GO, It is not possible with my PC to reduce less than 20ms (with 96KHz  sample rate) which is not acceptable when you mix real organ tube and simulation tubes...

Is it possible by using external sound card ?
 or increase the processor speed of the PC ? Dual core 3GHz ? 

Thank you.

PLEASE FIND MY SCREEN FILE !

**Comment by oleg68** [comment]
Please post the screenshot of your Settings->Audio tab.

**Comment by oleg68** [comment]
Please update your GrandOrgue version

**Reply by BenClam** [reply]
Merci beaucoup.
Avec la 3.6 de GO et le driver ASIOforAll cela fonctionne beaucoup mieux. J'arrive entre 7 et 17 ms en fonction de mes ordinateurs.

**Comment by midimusic** [comment]
@BenClam 
How are you measuring latency? I do not believe your figures. Real organ pipes  take anything up to 100 ms to get up to full volume!!
Technically there is no standard method to measure latency.
csw900
